mathicsscript/__main__.py

```diff
@@ -333,6 +333,12 @@ def main(
 
             evaluation = Evaluation(shell.definitions, output=TerminalOutput(shell))
             query, source_code = evaluation.parse_feeder_returning_code(shell)
+            if len(source_code) and source_code[0] == "!":
+                os.system(source_code[1:])
```

Comment:
We should probably place an option somewhere to disable this functionality, shouldn't we? For example, someone running mathics on a webserver wouldn't want to allow strangers to run arbitrary scripts on their machine.